src/pages/_docs/placeholder.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Placeholder
+menu: docs.components.placeholder
+description: Placeholder is used to reserve space for content that soon will appear in a layout.
+---
```

Comment:
Cool URLs never change, adding a rewrite is nice for end users who might have found an old link:
```suggestion
redirect_from:
    - /docs/skeleton.html
---
```

Reply:
sure, thank you!